tools/gsc-signature_score/signature_score.R

```diff
@@ -192,21 +177,16 @@ ggplot(data.frame(density_score[1:2]), aes(x, y, fill = ifelse(x < mean(score$sc
   )
 
 #Check patient distribution in two groups
-counts <- data.frame(table(metadata_filtered$Signature_category, metadata_filtered$Sample.name))
-
-ggplot(data = counts, aes(x = Var2, y = Freq, fill = Var1)) +
-  geom_bar(stat = "identity", position = position_dodge(), alpha = .8) +
-  scale_fill_manual(values=c("#ff0000", "#08661e")) +
-  labs(title = "Cell score distribution by patient", fill = "Score", x = "Patient")
+# counts <- data.frame(table(signature_output$rate, signature_output$cell))
 
 #Check score independant of low expression
-p_gene <- ggplot(metadata_filtered, aes(Signature_category, nGene)) +
-  geom_violin(aes(fill = Signature_category), alpha = .5, trim = F, show.legend = F) +
+p_gene <- ggplot(signature_output, aes(rate, nGenes)) +
+  geom_violin(aes(fill = rate), alpha = .5, trim = F, show.legend = F) +
   scale_fill_manual(values=c("#ff0000", "#08661e")) +
   geom_jitter() + labs(y = "Number of detected genes", x = "Signature")
 
-p_counts <- ggplot(metadata_filtered, aes(Signature_category, total_counts)) +
-  geom_violin(aes(fill = Signature_category), alpha = .5, trim = F, show.legend = F) +
+p_counts <- ggplot(signature_output, aes(rate, total_counts)) +
```

Comment:
I do not understand the title of this graph. What does that mean ?

Reply:
Ok pour "Influence of library sequencing depth on cell signature scores"